### ISSUE TITLE
`await` missing in async sample code

### ISSUE THREAD

**Issue by ryuheechul** [googleapis/gapic-generator-python#1912]
For example, this one
https://github.com/googleapis/google-cloud-python/blob/e6b5ea0a5b52ae56c944457c533b0ad225020357/packages/google-cloud-resource-manager/samples/generated_samples/cloudresourcemanager_v3_generated_folders_list_folders_async.py#L47

should be as below

```diff
-     page_result = client.list_folders(request=request)
+     page_result = await client.list_folders(request=request)
```

And it seems to be that many sample code are like that not just this one

**Comment by parthea** [comment]
The sample in https://github.com/googleapis/gapic-generator-python/issues/1912#issue-2091455570 is automatically generated using the code generator in this repository. See a [similar issue with the async samples](https://github.com/googleapis/gapic-generator-python/issues/1279 ) and [the corresponding fix](https://github.com/googleapis/gapic-generator-python/pull/1280/files) to get an idea of the type of fix that would be needed for this issue.

**Comment by kitokirti** [comment]
Hi @parthea , do we need to make changes in this particular file or all the files under generated_samples folder?
Please suggest.

**Comment by kitokirti** [comment]
So would you like me to do the `asyncio.run(...)` changes first? Please suggest.

**Comment by parthea** [comment]
Yes, you could try https://github.com/googleapis/gapic-generator-python/issues/1795 first
